examples/platform/silabs/efr32/wf200/host_if.cpp

```diff
@@ -532,9 +540,47 @@ static void wfx_events_task(void * p_arg)
         {
             if (!retryInProgress)
             {
+                retryInProgress = true;
+                if (wfx_is_sta_provisioned())
```

Comment:
```c
is_provisitioned = wfx_is_sta_provisioned(); // if you want to remove additional function call
SILABS_LOG("%s: wfx_is_sta_provisioned: %d", __func__, is_provisitioned);
// if you want to call function again
SILABS_LOG("%s: wfx_is_sta_provisioned: %d", __func__, wfx_is_sta_provisioned());
```

Reply:
Removed the logs because only for debugging purpose added the logs.